plugins/slack_app/config/routes.rb

```diff
@@ -0,0 +1,3 @@
+Samson::Application .routes.draw do
+  get '/slack/oauth', to: 'slack_app#oauth'
```

Comment:
maybe slack_app as route since the slack notification plugin might use slack already